apps/backend/src/libs/modules/file/file.ts

```diff
@@ -0,0 +1,7 @@
+import { config } from "../config/config.js";
+import { BaseFile } from "./file.module.js";
+
+const file = new BaseFile(config);
```

Comment:
```suggestion
const file = new File(config);
```
base only for implementation things